scripts/go-to-slide.js

```diff
@@ -21,8 +23,20 @@ H5P.CoursePresentation.GoToSlide = (function ($) {
       title = title ? title : cp.l10n.goToSlide.replace(':num', slideNum);
       classes += ' h5p-visible';
     }
+    
+    // Check if previous or next is selected.
+    if (goToSlideNextPrevious == "next") {
+      slideNum = index + 1;
+      // If the title wasn't set, you don't want slide undefined to be shown.
+      title = title.replace('slide undefined', "next slide");
+    } else if (goToSlideNextPrevious == "previous") {
+      slideNum = index - 1;
+      title = title.replace('slide undefined', "previous slide");
```

Comment:
Ahhh... I added a [new commit](https://github.com/h5p/h5p-course-presentation/pull/40/commits/bf563d2e2720e1ee946bbfb05bd23e567355d41a) to handle this.